docs/changelog.md

```diff
@@ -4,7 +4,7 @@
 
 # Changelog
 
-## unreleased
+## 0.62.0 (December 2024)
 
 - AerBackend now reject circuits with too many qubits
```

Comment:
minor thing... I think "rejects" fits better here than "reject"

Reply:
done in [d4d3c38](https://github.com/CQCL/pytket-qiskit/pull/441/commits/d4d3c385c0edba0d25f2fd68f1dc61725e563150)